src/Controls/src/Core/Shell/SearchHandler.cs

```diff
@@ -20,6 +20,9 @@ public class SearchHandler : BindableObject, ISearchHandlerController, IPlacehol
 
 		public event EventHandler<EventArgs> Focused;
 		public event EventHandler<EventArgs> Unfocused;
+		
+		internal event EventHandler<EventArgs> ShowKeyboardRequested;
```

Comment:
I wonder if these keyboard api should live in an interface so we can add to more controls , like entry for example .

Reply:
This is trick for now as it's not View.